.github/workflows/release.yml

```diff
@@ -0,0 +1,23 @@
+name: Release new version
+
+on: workflow_dispatch
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version-file: ".nvmrc"
+          registry-url: https://registry.npmjs.org/
+      - run: npm ci
+      - run: npm run lint
+      - run: npm run build
+      - name: Semantic Release
```

Comment:
Will it skip if previous steps fail?

Reply:
Lint and build should throw if failing, and same for ci (install)